foundations/azure/referenceImplementations/core/managementGroupTemplates/policyDefinitions/Compliant-APIManagementPolicySetDefinition.json

```diff
@@ -84,6 +84,328 @@
                         }
                     },
                     "name": "Deny-APIM-WithOutVnet"
+                },
+                {
+                    "properties": {
+                        "displayName": "API Management services should disable public network access",
+                        "mode": "Indexed",
+                        "description": "Azure API Management should disable public network access which provides enhanced security, isolation and allows you to prohibit public network traffic. The API service then only allows connections via the network connected via vnet injection or private endpoints.",
+                        "metadata": {
+                            "version": "1.0.1",
+                            "category": "API Management"
+                        },
+                        "parameters": {
+                            "effect": {
+                                "type": "String",
+                                "metadata": {
+                                    "displayName": "Effect",
+                                    "description": "Enable or disable the execution of the policy"
+                                },
+                                "allowedValues": [
+                                    "Audit",
+                                    "Deny",
+                                    "Disabled"
+                                ],
+                                "defaultValue": "Deny"
+                            },
+                            "evaluatedSkuNames": {
+                                "type": "Array",
+                                "metadata": {
+                                    "displayName": "API Management SKU Names",
+                                    "description": "List of API Management SKUs against which this policy will be evaluated."
+                                },
+                                "allowedValues": [
+                                    "Developer",
+                                    "Basic",
+                                    "Standard",
+                                    "Premium",
+                                    "Consumption"
+                                ],
+                                "defaultValue": [
```

Comment:
Why only a few SKUs as default? From a security perspective, it should apply to all SKUs assuming it is supported across all SKUs.

Reply:
Updated